deploy/olm-catalog/windows-machine-config-operator/manifests/wmc.openshift.io_windowsmachineconfigs_crd.yaml

```diff
@@ -59,6 +59,7 @@ spec:
             replicas:
               description: Replicas represent how many Windows nodes to be added to
                 the OpenShift cluster
+              format: int32
```

Comment:
You are sneaking this change in without clearly explaining why. Moreover why is this change in a commit that says `change folder structure for OLM manifests`. This requires a separate commit with a clear explanation linking to the issue requiring this change.